kod/object/passive/spell/radiusench/bane/qorbane.kod

```diff
@@ -0,0 +1,83 @@
+% Meridian 59, Copyright 1994-2012 Andrew Kirmse and Chris Kirmse.
+% All rights reserved.
+%
+% This software is distributed under a license that is described in
+% the LICENSE file that accompanies it.
+%
+% Meridian is a registered trademark.
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+QorBane is BaneEnchantment
+
+constants:
+
+   include blakston.khd
+
+resources:
+
+   include qorbane.lkod
+
+   QorBane_name_rsc = "Qorbane"
+   QorBane_icon_rsc = iqorbane.bgf
+   QorBane_desc_rsc = \
+      "Permeates an area with the holy essence of Shal'ille, harming any who "
+      "attempt to call on the evil energy of Qor. "
+      "Requires yrxl sap and fairy wings to cast."
+
+   QorBane_HurtCaster = "As you utter the final syllables of your spell, a "
+                        "wash of holy power passes through your body, "
```

Comment:
These resources should be formatted the same as resources in all the other classes (i.e. name_rsc = \ <newline> "text").